README.md

```diff
@@ -64,19 +64,21 @@ The build artifacts will be stored in the `dist/` directory.
 
 ### Running tests
 
-Before running the tests make sure that the project is built. To build the 
+Before running the tests make sure that the project is built. To build the
 project once you can use:
 
 ```bash
 ng build
 ```
 
-With the project built in the `dist/` folder you can just run: `karma start`. 
+With the project built in the `dist/` folder you can just run: `karma start`.
 Karma will run the tests and keep the browser open waiting to run again.
 
-This will be easier when the command 
+This will be easier when the command
 [ng test](https://github.com/angular/angular-cli/issues/70) is implemented.
 
+To run the end-to-end protractor tests, just do `$(npm bin)/protractor`.
```

Comment:
Please add a note saying that you need to have the server running (`ng serve`) to execute protractor tests?